cmd/viam-agent/main_windows.go

```diff
@@ -123,7 +118,8 @@ func zapChildren() error {
 		goutils.UncheckedError(elog.Info(1, fmt.Sprintf("KillTree stopping %d children of pid %d", len(lines), pid)))
 	}
 	for _, line := range lines[1:] {
-		if line == "" {
+		line = strings.TrimSpace(line)
+		if _, err := strconv.Atoi(line); err != nil {
```

Comment:
are you parsing the pid twice now?

Reply:
This is more just a test that we're getting JUST an integer from the line (as opposed to doing a regex or something on the line) as I was getting a lot of errors in the event log from the "not a valid childProcess line" below. Could be more elegant. But for something that fires only once each run, I didn't overthink it.